socorro/external/rabbitmq/crashstorage.py

```diff
@@ -170,7 +171,7 @@ def new_crashes(self):
         # queues the crash_id. The '_consume_acknowledgement_queue' function
         # is run to send acknowledgments back to RabbitMQ
         self._consume_acknowledgement_queue()
-        conn = self.rabbitmq.connection()
+        self.rabbitmq.connection()  # XXX is this ever used and needed?
```

Comment:
Ok, to test it I ran my local processor, removed the line and processing still works. I see stackwalking happening just fine. 
When I put in a `raise Exception` here, it did raise which means that this code definitely is used in my local process. 

I'm confident I can delete it now.